proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_sv_SE.properties

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Du är redan ansluten till denna servern!
```

Comment:
```suggestion
velocity.error.already-connected=Du är redan ansluten till denna server!
```
"Servern" would be used if you said "Den här servern" otherwise it sounds a little odd

https://sv.wiktionary.org/wiki/denna
> Efter denna används obestämd form och efter den här används bestämd form